### PR TITLE
fix: upgrade node, runtime and services

### PR DESCRIPTION
**Pull request by ivan-cholakov**
Upgraded the runtimes, the node, and the avn-service

### CODE REVIEW

File: primitives/avn-common/src/event_types.rs
Comment (thadouk) [inline primitives/avn-common/src/event_types.rs:1]: this file changes look incomplete.

File: primitives/avn-common/src/tests/test_event_types.rs [hunk 1]
```diff
@@ -398,14 +398,14 @@ fn test_added_validator_parse_bytes_long_topic() {
...
-    assert_eq!(expected, actual);
-}
+// #[test]
+// fn test_hashed() {
+//     let event_id = (H256::zero(), H256::zero());
+
+//     let actual = event_id.using_encoded(<T as frame_system::Config>::Hashing::hash)//event_id.hashed(BlakeTwo256::hash);
+//     let expected = BlakeTwo256::hash_of(&event_id);
+//     assert_eq!(expected, actual);
+// }
```
Comment (thadouk) [inline primitives/avn-common/src/tests/test_event_types.rs:408]: we should restore this test.

File: primitives/avn-common/Cargo.toml
Comment (thadouk) [inline primitives/avn-common/Cargo.toml:1]: it feels like we pulled in primitive changes in this PR which is out of scope

**Review by thadouk — COMMENTED** [review]
I think this PR needs a bit refinement before reviewing and merging.

File: primitives/avn-common/Cargo.toml
Comment (ivan-cholakov) [reply primitives/avn-common/Cargo.toml:1]: Not really, I created this branch with node, runtime and primitives

File: runtime/common/Cargo.toml [hunk 2]
```diff
@@ -11,16 +11,18 @@ repository = { workspace = true }
 
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
-hex-literal = { version = "0.3.4", optional = true }
+# codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = ["derive"] }
```
Comment (thadouk) [inline runtime/common/Cargo.toml:14]:
```suggestion
```

File: runtime/test/Cargo.toml [hunk 3]
```diff
@@ -14,78 +14,80 @@ rust-version = { workspace = true }
 targets = ["x86_64-unknown-linux-gnu"]
 
 [build-dependencies]
-substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36" }
+substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v1.0.0", optional = true}
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
-hex-literal = { version = "0.3.4", optional = true }
+# codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = ["derive"] }
```
Comment (thadouk) [inline runtime/test/Cargo.toml:20]:
```suggestion
```

File: runtime/test/Cargo.toml [hunk 4]
```diff
@@ -221,7 +231,7 @@ try-runtime = [
 	"pallet-aura/try-runtime",
 	"pallet-authorship/try-runtime",
 	"pallet-balances/try-runtime",
-	"pallet-collator-selection/try-runtime",
+	# "pallet-collator-selection/try-runtime",
```
Comment (thadouk) [inline runtime/test/Cargo.toml:234]:
```suggestion
```

File: runtime/test/src/lib.rs [hunk 5]
```diff
@@ -171,35 +170,11 @@ const STORAGE_ITEM_NAMES: &[&'static str] = &[
 pub struct SeedBridgeTransactionAndDropEthTransactions;
 
 impl frame_support::traits::OnRuntimeUpgrade for SeedBridgeTransactionAndDropEthTransactions {
```
Comment (thadouk) [inline runtime/test/src/lib.rs:172]: This looks incomplete.
Probably easier if we drop the `SeedBridgeTransactionAndDropEthTransactions` migration

File: runtime/test/src/xcm_config.rs
Comment (thadouk) [inline runtime/test/src/xcm_config.rs:1]: This file looks incomplete. Should mimic the one from avn runtime

**Review by thadouk — APPROVED** [review]
